tree/tree/src/TBasket.cxx

```diff
@@ -1329,9 +1356,8 @@ Int_t TBasket::WriteBuffer()
          noutot += nout;
          objbuf += kMAXZIPBUF;
          nzip   += kMAXZIPBUF;
-         if (!precisionCascades.empty()) {
-            // increment each element by its corresponding nout.
-         }
+         for (size_t i = 0; i<precisionCascades.size(); i++)  // does nothing if no precision cascade
+            precisionCascades[i] += nouts[i];  // increment each element by its corresponding nout.
```

Comment:
```suggestion
            precisionCascades[i] += nouts[i + 1];  // increment each element by its corresponding nout.
```

Reply:
I don't understand the suggestion to look at the nouts value from one ahead. The first thing I do with precisionCascades is push_back the primary before, so both it and nouts have the primary buffer first and there should be no offset between them.

Reply:
yes, I forgot that early push, so you are right.